kit/inputs/src/MaterialTextView/hooks/useApplyMask/amount/runUIFormat.ts

```diff
@@ -61,7 +61,7 @@ export function runUIFormat(
             normalizedDecimalPart = normalizedDecimalPart.slice(0, countOfDecimalDigits);
         }
 
-        normalizedText += delimeter;
+        normalizedText += '.';
```

Comment:
I don't think that it should be a dot symbol here (as the delimiter depends on the localization)

Reply:
This has been changed for testing purposes. I just forgot to undo these changes.